app/component/map/map.cjsx

```diff
@@ -53,10 +42,12 @@ class Map extends React.Component
       paddingTopLeft: props.padding)
 
   componentDidMount: ->
-    @context.getStore('PositionStore').addChangeListener @onChange
     @context.getStore('EndpointStore').addChangeListener @onChange
     L.control.attribution(position: 'bottomleft', prefix: false).addTo @refs.map.getLeafletElement()
-    if not (@props.disableZoom or L.Browser.touch)
+    if @props.disableMapTrack
+      @refs.map.getLeafletElement().addEventListener('dragstart', @props.disableMapTrack)
+      @refs.map.getLeafletElement().addEventListener('zoomend', @props.disableMapTrack)
+    if not @props.disableZoom or L.Browser.touch
       L.control.zoom(position: 'topleft').addTo @refs.map.getLeafletElement()
     if @props.fitBounds
       @setBounds(@props)
```

Comment:
These could be added as props to the map component in the render function as defined in https://github.com/PaulLeCam/react-leaflet/blob/master/src/Map.js#L19-L20